base/refpointer.jl

```diff
@@ -70,3 +70,10 @@ setindex!(b::RefValue, x) = (b.x = x; b)
 setindex!(b::RefArray, x) = (b.x[b.i] = x; b)
 
 ###
+
+function showcompact(io::IO, b::RefValue)
```

Comment:
To clarify, sometimes `showcompact(::Array)` is bigger than `show(::Array)` because `showcompact` seems to do a `showall` even though it skips the formatting.